lib/assets_precompile_enforcer/sprockets/rails/helper.rb

```diff
@@ -0,0 +1,59 @@
+require 'sprockets/rails/helper'
```

Comment:
Are the contents of this file the same as `sprockets/helpers/rails_helper.rb`, just with a different name for different Rails versions? Or are there differences?